minimesos/src/main/java/com/containersol/minimesos/MesosCluster.java

```diff
@@ -198,7 +310,7 @@ private static void destroyContainers(String clusterId) {
     }
 
     public List<AbstractContainer> getContainers() {
-        return containers;
+        return Collections.unmodifiableList(containers);
```

Comment:
Sometimes I also try to protect internal collections, but normally it only complicates things later. Can we return it 'as-is'?